Assets/Scripts/PlayerSupervisor.cs

```diff
@@ -98,25 +109,10 @@ void Start()
     void CountBlocks()
     {
         activeBlocks = 0;
-
-        // TODO need to count only blocks in each player's environment to
-        //  work for general multiplayer use; Maybe keep all blocks in a
-        //  container for easier tracking
-        if (gameManager.trainingMode)
+        foreach (Block block in FindObjectsOfType<Block>())
         {
-            foreach(Transform child in trainingBlocksInstance.transform)
-            {
-                if (child.gameObject.GetComponent<Block>() && child.gameObject.activeSelf)
-                    ++activeBlocks;
-            }
-        }
-        else
-        {
-            foreach (Block block in FindObjectsOfType<Block>())
-            {
-                if (block.gameObject.activeSelf)
-                    ++activeBlocks;
-            }
+            if (block.gameObject.activeSelf)
+                ++activeBlocks;
```

Comment:
The trouble with counting the blocks this way is that it will count every block in the scene, rather than only the blocks in the instance. Only the blocks in the block container should be counted.

Reply:
good catch, fixed this, back to modified if/else statement that was in there prior to me changing it. first direction will count blocks for multi window training, and second will count blocks in single window training.